scripts/deploy.sh

```
@@ -0,0 +1,215 @@
+        # Build Docker image
+        echo "Building Docker image..."
+        docker build -t ch11-bankruptcy-api:latest .
+        
+        # Get ECR repository URL if not set
+        if [ -z "$ECR_REPO" ]; then
+            ECR_REPO=$(aws ecr describe-repositories --repository-names ch11-bankruptcy-api --query 'repositories[0].repositoryUri' --output text)
+        fi
+        
+        # Login to ECR
+        echo "Logging in to ECR..."
+        aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $ECR_REPO
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)